lib/jekyll-admin/server/collection.rb

```diff
@@ -47,26 +49,70 @@ def collection
         collection[1] if collection
       end
 
+      def request_path
+        sanitized_path File.join(site.source, request_payload["path"])
```

Comment:
Option 1: Add the slash `path << '/' unless path.end_with?('/')`

Option 2: Remove the slash `path.sub(%r{/$}, '')`